Description: Fix wxGTK slider label update
 Make sure value label updates even if handle position does not change.
Author: PC
Bug: http://trac.wxwidgets.org/ticket/13042
Applied-Upstream: http://trac.wxwidgets.org/changeset/67379
Last-Update: 2012-05-31

Index: wxWidgets/branches/WX_2_8_BRANCH/src/gtk/slider.cpp
===================================================================
--- a/src/gtk/slider.cpp (revision 44603)
+++ b/src/gtk/slider.cpp (revision 67379)
@@ -346,4 +346,6 @@
         BlockScrollEvent();
         gtk_range_set_value(GTK_RANGE (m_widget), value);
+        // GTK only updates value label if handle moves at least 1 pixel
+        gtk_widget_queue_draw(m_widget);
         UnblockScrollEvent();
     }
